### PR TITLE
W-11427407 update postActionGroup since the parent class in file common got updated

### PR DESCRIPTION
**Pull request by rogerwang1102**
update postActionGroup since the parent class in file common got updated

### CODE REVIEW

**Review by acurci-at-mulesoft — CHANGES_REQUESTED** [review]
Could you add a test suite to give coverage to the new feature?

**Comment by rogerwang1102** [comment]
OK.

File: src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java [hunk 1]
```diff
@@ -6,7 +6,28 @@
  */
 package org.mule.extension.ftp;
 
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 import static org.mule.extension.ftp.AllureConstants.FtpFeature.FTP_EXTENSION;
+
+import java.nio.file.Path;
+import java.util.LinkedList;
+import java.util.Queue;
```
Comment (acurci-at-mulesoft) [inline src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java:15]: Move these imports after the mule imports.
Remember the order of the imports is 
mule
java 
others

File: src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java [hunk 2]
```diff
@@ -40,4 +63,194 @@ public void deleteAndMove() {
...
+  @Test(expected = FileAlreadyExistsException.class)
+  @Description("verifies that move file with overwrite flag set to false and exceptiopn will be thrown")
+  public void testMoveWithoutOverwriteWhenFileExists() {
+    FileAttributes fileAttributes = mock(FileAttributes.class);
+    when(fileAttributes.getName()).thenReturn("");
+    FileConnectorConfig fileConnectorConfig = mock(FileConnectorConfig.class);
+
+    ConcreteFileSystem fileSystem = new ConcreteFileSystem("");
+
+    new PostActionGroupChild("someDir", null, false, false).apply(fileSystem, fileAttributes, fileConnectorConfig);
```
Comment (acurci-at-mulesoft) [inline src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java:77]: Tests should include assertions. Add at least one assertion to this test case

**Review by acurci-at-mulesoft — CHANGES_REQUESTED** [review]

File: src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java (hunk 2 above)
Comment (rogerwang1102) [reply src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java:77]: This is to test exception got thrown correctly. So I think no assertion here.
This kind of test case also exists in file common.

Comment (acurci-at-mulesoft) [reply src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java:77]: Sorry, I wasn't clear, I mean that you should put something like this... 
`fail("Expected exception was not thrown");`
but no worry. I approve the ticket. It is not important

**Review by acurci-at-mulesoft — APPROVED** [review]

**Pull request merged**